### DISCUSSION TITLE
unity 6

### DISCUSSION THREAD

**Discussion by ElliotHYLee** [jeffreylanters/react-unity-webgl#564] [Feature Requests]
Hi, I wonder if there's any specific configureation or update plan for unity 6?

**Comment by GeraldOB1988** [comment]
> Hi, I wonder if there's any specific configureation or update plan for unity 6?

I was wondering the same ! I have been having issues with sendMessage.

**Reply by jeffreylanters** [reply]
That issues are you running into? I might be able to help you resolve these.

**Reply by ElliotHYLee** [reply]
Yup, can't send the message.

**Comment by jeffreylanters** [comment]
There shouldn't be any changed as far as I'm aware of. But I'm going to look into it asap, I'll keep you posted on this thread.

**Comment by mrandrewmclean** [comment]
I'm having a nightmare with Unity 6 and WebGL. It worked perfectly before but any U6 webgl seems to freeze the tab if a new tab is opened (off focus). I can't even get it to debug - the freezing happens so quickly. I've spent a day trying every variation of WebGL settings - same thing. Reverting back to 2022, works fine. Anyone else seeing this? Driving me mad.
